6/README.md

```diff
@@ -0,0 +1,275 @@
+---
+domain: rfc.tango-controls.org
+shortname: 6/Database
+name:Database
+status: draft
+editor: 
+contributors: [David Erb](mailto:david.erb@maxiv.lu.se)
+---
+
+This document describes the Tango Database specification.
+
+## Preamble
+
+Copyright (c) 2019 Tango Controls
+
+This Specification is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version. This Specification is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with this program; if not, see <http://www.gnu.org/licenses>.
+
+This Specification is a [free and open standard](http://www.digistan.org/open-standard:definition) and is governed by the Digital Standards Organization's [Consensus-Oriented Specification System](http://www.digistan.org/spec:1/COSS).
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Related RFC specifications (TBD)
+- X/Device Model 
+- X/Attributes 
+- X/Properties 
+
+## Tango Database specification
+
+This specification is intended to formally document the Tango Database. 
+Runtime implementations of the Database in conforming Tango implementations MUST follow this specification.
+
+
+### Goals of the Database
+
+* provide means for device servers to be configured from central control
+* serves as nameserver such that clients are able to find a device server's network location at runtime by name
+
+### Use Cases of the Database
+
+* stores configuration data (properties) used at startup of a device server 
+* acts as nameserver by storing the dynamic network addresses
+* acts as permanent store of dynamic settings which need to be memorised
+* ensures the uniqueness of device name and of aliases
+* keeps list of controlled servers for Starter service
+* limited set functionality provide by file as alternative to network database
+
+
+### General Specification
+The database service shall be implemented as a Tango Device Server.
```

Comment:
so call Tango Database (or Database or Database device if you prefer).
You can this term instead of using database service, database device, ...